lib/dialogs/import/source-importer/executor/index.tsx

```diff
@@ -115,7 +115,7 @@ class ImportExecutor extends Component<Props> {
               disabled={locked}
             />
           </label>
-          {hint && <p className="theme-color-fg-dim">{hint}</p>}
+          {hint && <p className="hint">{hint}</p>}
```

Comment:
I assume the CSS for hint exists elsewhere?

Reply:
I actually don't think this hint is used anymore at all so never gets displayed. I added the class `hint` just in case we needed to style it in the future. But I believe it could be removed altogether.

Reply:
PR to remove it here https://github.com/Automattic/simplenote-electron/pull/3027